timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue

```diff
@@ -0,0 +1,101 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<template>
+  <div :style="expanded ? 'background-color: #EEEEEE' : ''">
```

Comment:
How does this play with the theming (dark mode?)

Reply:
Good catch, I fixed it to now also work well with the dark theme.